sbin/root-mount

```diff
@@ -130,7 +133,7 @@ factory_reset_if_needed()
 
 		YAMUIPID=$!
 
-		if factory-reset-lvm $LVM_ROOT_SIZE $LVM_RESERVED_MB; then
+		if factory-reset-lvm $LVM_ROOT_SIZE $LVM_RESERVED_MB $WIPE; then
```

Comment:
Nowhere, it was defined as a command line argument to factory-reset-lvm where SAILFISHOS_WIPE_PARTITIONS is now exported.  

It didn't seem practical to user a command line argument for the flash-partition script though because there's a wrapper script for every partition that would also have to be modified.